Client/Client.Qt/Widgets/MessageWidget.cpp

```diff
@@ -1,274 +1,129 @@
 ﻿#include "MessageWidget.hpp"
 
+#include <QPainter>
+#include <QtEvents>
+#include <utility>
+
+#include "ChatHistory.hpp"
 #include "Style/Style.hpp"
```

Comment:
Place the headers in some order, but not in a jumble.

Reply:
Clang Format